web-console/src/lib/components/pipelines/editor/TabChangeStream.svelte

```diff
@@ -47,8 +47,8 @@
         },
         {
           paths: ['$.json_data.*'],
-          bufferSize: 1 * 1024 * 1024,
-          separator: '\n'
+          bufferSize: 8 * 1024 * 1024,
```

Comment:
Is increasing the buffer size the fix for https://github.com/feldera/feldera/issues/2757 ? What will happen when the new buffer size is reached? Is there a message pop-up informing web-console cannot keep up with the change stream? Similarly, this is 1MB of buffer size already, whereas for the linked issue it should not get close to that amount.

Reply:
No, increase to 8 MiB is unrelated to the fix. In my testing I found this to be a better value to allow spikes in throughput to be processed reasonably without skipping bytes and affecting peformance